addons/web/static/src/search/control_panel/control_panel.js

```diff
@@ -115,7 +115,7 @@ export class ControlPanel extends Component {
             embeddedInfos: {
                 showEmbedded:
                     this.env.config.embeddedActions?.length > 0 &&
-                    (!!this.env.config.parentActionId ||
+                    ((!!this.env.config.parentActionId && !!JSON.parse(browser.localStorage.getItem('showEmbeddedItems')) )||
```

Comment:
Why this name "EmbeddedItems" where we talk about "EmbeddedActions" ? :thinking: